metaflow/metaflow_environment.py

```diff
@@ -55,6 +55,13 @@ def add_to_package(self):
         """
         return []
 
+    # TODO: this must be unified as '1 linter config' for pylint, ruff
+    def ruff_config(self):
```

Comment:
We are explicitly passing the config from `ruff.toml` situated at `metaflow/linters/ruff/ruff.toml`. Alternatively, we can have this default config here, but a [toml is a preferred/recommended](https://docs.astral.sh/ruff/configuration/) manner.